checkstyle.xml

```diff
@@ -112,5 +112,12 @@
 			<property name="message" value="Don't use System.out/err, use LOG4J2 instead."/>
 			<property name="ignoreComments" value="true" />
 		</module>
+		<module name="MatchXpath">
```

Comment:
Dit resulteert in IntelliJ in de volgende melding: `Single line comment text should start with space. (410:25) [MatchXpath]`. 

Ik heb het nu als 'warning' toegevoegd zodat het de build niet breekt.